src/main/kotlin/id/walt/services/essif/userwallet/UserWalletService.kt

```diff
@@ -46,7 +47,7 @@ object UserWalletService {
 //        DidService.create(DidMethod.web)
 //    }
 
-    private val credentialService = VCService.getService()
+    private val credentialService = JsonLdCredentialService.getService()
```

Comment:
I think the implementation to use should be configurable. Furthermore, ESSIF/EBSI only supports JWT Proof for now